pallets/eth-bridge/src/lib.rs

```diff
@@ -219,9 +223,13 @@ pub mod pallet {
                 BoundedVec<(BoundedVec<u8, TypeLimit>, BoundedVec<u8, ValueLimit>), ParamsLimit>,
             caller_id: BoundedVec<u8, CallerIdLimit>,
         },
-        /// EventProcessed(bool, EthEventId)
-        EventProcessed {
-            accepted: bool,
+        EventAccepted {
+            eth_event_id: EthEventId,
+        },
+        EventRejected {
+            eth_event_id: EthEventId,
+        },
```

Comment:
why use two events for signaling the processing events?

Reply:
Would it not be due to the fact that there are two possibilities in the event being accepted vs rejected and to distinguish between these two circumstances?

Reply:
We have connected components that are currently looking for these events and its a lesser change to maintain the setup. Its also easier to search and filter instead of using event args.